index.html

```diff
@@ -1,22 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@200;400;600&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@200;400;600&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
     <link rel="stylesheet" href="style.css" />
   </head>
+
   <body>
     <header class="navbar">
       <nav>
```

Comment:
- [ ] As requested by the previous reviewer, kindly try to match the Figma design as much as possible for the header. Please make the spaces on the left and right equal. Right now, there is some space before the logo but there is no space after the links. The logo is also breaking in 2 lines.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199701951-3c1823ed-a8db-451f-8a70-6a5b1a3d3675.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199701993-0beebfb9-d337-408f-90a3-8f12205c1ef7.png)